WordPress/Classes/ViewRelated/Blog/Site Picker/SitePickerViewController+SiteActions.swift

```diff
@@ -25,7 +25,6 @@ extension SitePickerViewController {
         let menuItems = [
             MenuItem.visitSite({ [weak self] in self?.visitSiteTapped() }),
             MenuItem.shareSite { [weak self] in self?.buttonShareSiteTapped() },
-            MenuItem.addSite({ [weak self] in self?.addSiteTapped()} ),
```

Comment:
This menu is available when long-pressing the site picker dropdown on the dashboard. It's a somewhat hidden feature that was added during a hack week, and we don't officially support it. So, it's safe to remove this option. Almost nobody knows this menu exists. Originally, the plan was to add this menu as a separate button, but it didn't materialize.